openml/datasets/dataset.py

```diff
@@ -127,8 +128,8 @@ def __init__(
         update_comment=None,
         md5_checksum=None,
         data_file=None,
-        features=None,
-        qualities=None,
+        features_file: Optional[str] = None,
+        qualities_file: Optional[str] = None,
```

Comment:
I would like to not do it in this PR as it might require adapting a lot of function calls here and there. I only changed these two as I specifically worked on them, and adding these two type annotations already required some changes.

Reply:
Fair enough